cypress/integration/version/task_table.ts

```diff
@@ -18,6 +18,9 @@ describe("Task table", () => {
   it("Updates the url when column headers are clicked", () => {
     cy.visit(pathTasks);
     waitForTaskTable();
+    // TODO: Remove wait in DEVPROD-597.
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(200);
```

Comment:
I don't like the wait but given how frustrating it has been for the team experiencing this flake, I agree that it's okay as a temporary solution, since we are getting rid of the antd tables soon.